ui-admin/src/navbar/AdminSidebar.tsx

```diff
@@ -29,7 +30,15 @@ const AdminSidebar = () => {
   }
   const currentStudy = studyList.find(study => study.shortcode === studyShortcode)
 
-  return <div style={{ backgroundColor: '#333F52', minHeight: '100vh', minWidth: open ? '250px' : '50px' }}
+  const color =
+    config.deploymentZone === 'local'
```

Comment:
this will be cleaner with a color map instead of a ternary. e.g.
```
const ZONE_COLORS = { demo: ..., local: .. }
const color = ZONE_COLORS[config.deploymentZone] ?? ZONE_COLORS['prod']
```

Reply:
Love it, much better than the ugly ternary